example/dags/snowflake_sample_dag.py

```diff
@@ -69,15 +69,23 @@
 SUPPORTED_SCHEMA_SQL_IN_CLAUSE = "('{schemas}')".format(schemas="', '".join(SUPPORTED_SCHEMAS))
 
 # SNOWFLAKE CONFIGs
-SNOWFLAKE_DATABASE_KEY = 'YOUR_SNOWFLAKE_DB_NAME'
+SNOWFLAKE_DATABASE = 'YOUR_SNOWFLAKE_DB_NAME'
```

Comment:
why we make this change when it still uses https://github.com/amundsen-io/amundsendatabuilder/blob/804b20cf0c9804a4b47760d740e6a1bb8e1dc33c/databuilder/extractor/snowflake_metadata_extractor.py#L60

Reply:
> and has this been tested?

Yes, I tested the sample scripts and found this issue. and pushing this PR as a fix.

Reply:
Yeah. I think we went back and forth on the other PR. Let me just use what is in `snowflake_metadata_extractor.py` to keep them consistent for now.

Reply:
sounds good, yeap, let's keep it consistent.